iotdb-core/datanode/src/main/java/org/apache/iotdb/db/tools/validate/TsFileValidationTool.java

```diff
@@ -579,12 +209,7 @@ private static boolean checkArgs(String[] args) {
   }
 
   public static void clearMap(boolean resetBadFileNum) {
```

Comment:
Maybe validationScan should be cleaned after scanning all files in one time partition?

Reply:
Maybe we want to detect cross-partition overlap?